mmv1/products/appengine/FlexibleAppVersion.yaml

```diff
@@ -139,6 +139,14 @@ properties:
         description: |
           List of ports, or port pairs, to forward from the virtual machine to the application container.
         item_type: Api::Type::String
+      - !ruby/object:Api::Type::Enum
+        name: 'instanceIpMode'
+        description: |
+          Prevent instances from receiving an ephemeral external IP address.
+        min_version: beta
+        values:
+          - :external
```

Comment:
EXTERNAL/INTERNAL per https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1beta/apps.services.versions#instanceipmode

The API accepts these formats but returns the uppercase ones.

Reply:
OK let's try; will switch to uppercase.

Reply:
Updated